components/engine_rocks/src/raft_engine.rs

```diff
@@ -252,6 +252,8 @@ impl RaftEngine for RocksEngine {
         batch: &mut Self::LogBatch,
     ) -> Result<()> {
         batch.delete(&keys::raft_state_key(raft_group_id))?;
+        batch.delete(&keys::region_state_key(raft_group_id))?;
```

Comment:
Sometimes we need to keep tombstone regions, maybe region states should not be cleaned here in this case.

Reply:
`Tombstone` region state is put again after being cleaned.